backend/src/v2/driver/driver.go

```diff
@@ -313,6 +313,12 @@ func makePodSpecPatch(
 		return "", fmt.Errorf("failed to make podSpecPatch: %w", err)
 	}
 
+	// Convert environment variables
+	userEnvVar := make([]k8score.EnvVar, 0)
+	for _, envVar := range container.Env {
+		userEnvVar = append(userEnvVar, k8score.EnvVar{Name: envVar.Name, Value: envVar.Value})
```

Comment:
NIT: similarly, can we use `.GetName()` and `.GetValue()` instead?

Reply:
Similarly, see [reply](https://github.com/kubeflow/pipelines/pull/8743#discussion_r1083062553).